.gitignore

```diff
@@ -41,6 +41,9 @@ output/*/index.html
 docs/_build
 docs/modules.rst
 
+# LevelDB
+evm/db/backends/db
```

Comment:
So, `./evm` is the base source path for the `evm` python library.  That path is not an appropriate place for ephemeral files or installation specific files like a leveldb database.

After thinking this through, I don't think that the db path have a global default because there is no global default that is appropriate for all situations.  (see below).  This implies that the value should always be required.

Now there are some cases where default locations make sense.

* For testing:  Use the `tempdir` fixture that comes built in with pytest. See [this code in populus](https://github.com/pipermerriam/populus/blob/master/conftest.py#L34) for an example of how to use this fixture.
* For *production* chains, aka the mainnet, rinkeby, ropsten, use `$HOME/.pyevm/<chain-name>/chain-db`.  (this convention has already been established by the parity client as a location for the chain data.).

Worth pointing out that the go-ethereum client uses a slightly different convention, but I'm not inclined to follow their convention as the parity one seems more intuitive.